tests/Badge/Model/UseCase/CreateErroreBadgeTest.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Tests\Badge\Model\UseCase;
+
+use App\Badge\Model\Badge;
+use App\Badge\Model\UseCase\CreateErrorBadge;
+use GuzzleHttp\Exception\BadResponseException;
+use PHPUnit\Framework\TestCase;
+
+class CreateErroreBadgeTest extends TestCase
+{
+    public function testCreateAGenericErrorBadge(): void
+    {
+        $error = new \Exception('msg');
+        $format = 'svg';
+
+        $createErrorBadge = new CreateErrorBadge();
+
+        $badge = $createErrorBadge->createErrorBadge($error, $format);
+
+        $this->assertEquals(new Badge((string) $error, 'generic', 'e05d44', $format), $badge);
```

Comment:
`self::assertEquals()`?

Reply:
Why? Isn't this irrelevant?

Reply:
Yes! But it's a new best practice! No? :)

Reply:
> A common question, especially from developers new to PHPUnit, is whether using $this->assertTrue() or self::assertTrue(), for instance, is "the right way" to invoke an assertion. The short answer is: there is no right way. And there is no wrong way, either. It is a matter of personal preference.

Source: https://phpunit.de/manual/6.5/en/appendixes.assertions.html (Appendix A. Assertions)

Reply:
Yep! I know! But for me there is a micro-optimization to use a method static! 
#my2cent

I leave the choice to you! ;)